oonipipeline/src/oonipipeline/analysis/datasources.py

```diff
@@ -51,7 +57,31 @@ def iter_web_observations(
             last_msmt_uid = row[msmt_uid_idx]
             obs_group = []
 
-        obs_group.append(WebObservation(*row))
+        # TODO(art): this is super sketchy.
+        # We need to do this in order to obtain the correct offsets into the queried columns
```

Comment:
TODO(art): add either a comment or make it less sketchy

Reply:
I enriched this with a comment and filed this issue: https://github.com/ooni/data/issues/77